### PR TITLE
BaSP-TG-29: Create seeds of employee & super admins

### PR DESCRIPTION
**Pull request by JulianFloresDev**
Generate seeds files to :

- Employee
- Super Admins

### CODE REVIEW

File: src/seeds/superAdmins.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+export default [
+  {
+    _id: mongoose.Types.ObjectId('6352d89541876d1b1b99dbf6'),
+    name: 'Ignacio',
+    lastName: 'Aristo',
+    email: 'NoMeDigan_Nacho@gmail.com',
```
Comment (DinoStampella) [inline src/seeds/superAdmins.js:8]: Emails validation allow capital letters?

File: src/seeds/superAdmins.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    password: 'elMrSatanSinRulos',
+  },
+  {
+    _id: mongoose.Types.ObjectId('6352d7e777ae480815a63af2'),
+    name: 'Verona',
+    lastName: 'Baudracco',
+    email: 'ReDesaparecidaLaVero@gmail.com',
```
Comment (DinoStampella) [inline src/seeds/superAdmins.js:15]: Same as above

**Review by DinoStampella — CHANGES_REQUESTED** [review]

File: src/seeds/superAdmins.js (hunk 1 above)
Comment (JulianFloresDev) [reply src/seeds/superAdmins.js:8]: I think it shouldn't be like this, but it was for testing purposes only.

I'll check on Joi validations!

Thanks for comment

File: src/seeds/superAdmins.js (hunk 2 above)
Comment (JulianFloresDev) [reply src/seeds/superAdmins.js:15]: Reply

**Review by DinoStampella — APPROVED** [review]

**Review by GuidoG95 — APPROVED** [review]

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**